subsys/settings/Kconfig

```
@@ -109,6 +110,18 @@ config SETTINGS_CUSTOM
 	help
 	  Use a custom settings storage back-end.
 
+config SETTINGS_ITS
```

Suggest rather using
```suggestion
config SETTINGS_TFM_ITS
```
because just ITS could be more than just TF-M's.